### ISSUE TITLE
Wrong catalog and schema using dbt-project-evaluator with dbt Cloud and Databricks Unity Catalog

### ISSUE THREAD

**Issue by fuselessmatt** [dbt-labs/dbt-project-evaluator#331]
### Describe the bug

I tried set this up for out dbt instance on dbt Cloud and Databricks with Unity Catalog by [following the official guide](https://dbt-labs.github.io/dbt-project-evaluator/0.5/#using-this-package) and no other configurations. When I try to run the command as a dbt Cloud Job, two errors occur
1. The tables end up in the default catalog/database and schema as defined in the connection and environment setting. There was not option to set it to another catalog or schema similarly to how you do that in the rest of the dbt project (in with the `catalog` and `schema` keywords in the models section of `dbt_project.yml`)
2. `stg_nodes` specify `database`/catalog and `schema` as their default values instead of their real values.

### Steps to reproduce

Install the package and run the command `dbt build --select package:dbt_project_evaluator !` as dbt cloud Job

### Expected results

1. There should be a way to specify where the tables and up
2. The stg_nodes should display the actual catalog and schema and not just the default

### System information
**The contents of your `packages.yml` file:**
```yaml
packages:
  - package: calogica/dbt_date
    version: 0.7.2
  - package: dbt-labs/dbt_utils
    version: 1.0.0
  - package: dbt-labs/dbt_project_evaluator
    version: 0.5.0
```

**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [ ] snowflake
- [x] other (Databricks)

**The output of `dbt --version`:**
```
06:49:54  Running with dbt=1.4.6
06:49:54  Unable to do partial parsing because saved manifest not found. Starting full parse.
06:50:00  Found 253 models, 82 tests, 2 snapshots, 1 analysis, 638 macros, 0 operations, 3 seed files, 181 sources, 18 exposures, 5 metrics
06:50:00  The selection criterion '!' does not match any nodes
```

### Additional context

This beyond my dbt knowledge

### Are you interested in contributing the fix?

Tell me if there are any options or configuration you want to try and I can run them

**Comment by b-per** [comment]
Hi! Could you please show the content of the `dbt_project.yml` file of your main project here (or send it to me via DM on the community Slack (Benoit (dbt Labs))?

I quickly looked at the PR and saw this comment:

> This can be done in the `dbt_project.yaml` file of the main project or the `dbt_project.yaml` file of the installed package.
Changes in the main project's file will override values and installed package's file.

You should not modify the `dbt_project.yml` for another package, just the main one for your project.

**Comment by fuselessmatt** [comment]
We continued the discussion in Slack. In short everything works as expected if you specify the destination in the `dbt_project.yaml` of the main project. I updated https://github.com/dbt-labs/dbt-project-evaluator/pull/333 base on the feedback from the previous reply. Having this documented would have helped me determine where the problem was and will potentially help future clueless dbt users.

Unrelated to this plug-in someone in our team had developed a custom `get_custom_schema` macro that overrode the real destination. That is why it didn't look like it worked when I tried everything before.
